Common/include/option_structure.hpp

```diff
@@ -958,6 +956,20 @@ inline TURB_FAMILY TurbModelFamily(TURB_MODEL model) {
   return TURB_FAMILY::NONE;
 }
 
+/*!
+ * \brief SST Base Model
+ */
+enum class SST_BASE {
```

Comment:
Im not sure how the actual code/ enum list would look like in EnumList for the config.  Would we need an option for each option